### ISSUE TITLE
Create stream processor to export events

### ISSUE THREAD

**Issue by npepinpe** [zeebe-io/zeebe#1171]
## Criterias

- A new stream processor is created which forwards events of a single stream to a list of exporters.
- The stream processor keeps track of the position of the last event to be successfully exported for each exporter.
- Periodically write an event to the stream containing the list of exporters and their positions.

### Notes

Writing an event is a good solution to rebuild the state (useful while we don't replicate RocksDB state), and because exporting is considered a side effect, which even when we run processors in read-only mode on followers, will not be executed (since it's non deterministic).

To minimize the impact on the log stream, we can write batched updates (i.e. commit the positions every 10,000 events exported or some other metric, we can always tweak that later).

Blocked by #1170

**Comment by menski** [comment]
Writing the current state to the log is not part of this issue and will be implemented with #1214
